apple/RNSVGRenderable.mm

```diff
@@ -245,10 +245,17 @@ - (void)renderTo:(CGContextRef)context rect:(CGRect)rect
     CGSize boundsSize = bounds.size;
     CGFloat height = boundsSize.height;
     CGFloat width = boundsSize.width;
+    CGFloat scale = 0.0;
```

Comment:
I've initialized `scale` to `0.0` to suppress this warning: 

![CleanShot 2023-12-06 at 12 44 09@2x](https://github.com/software-mansion/react-native-svg/assets/52801365/419fd537-2a72-4673-a3b6-15b3939cc378)